src/main/java/net/dv8tion/jda/core/events/DisconnectEvent.java

```diff
@@ -17,14 +17,17 @@
 
 import com.neovisionaries.ws.client.WebSocketFrame;
 import net.dv8tion.jda.core.JDA;
+import net.dv8tion.jda.core.requests.WebSocketClient;
 
 import java.time.OffsetDateTime;
 
 /**
- * <b><u>DisconnectEvent</u></b><br>
- * Fired if our connection to the WebSocket was disrupted.<br>
- * <br>
- * Use: Reconnect manually or stop background threads that need fired events to function properly.
+ * Indicates that JDA has been disconnected from the remote server.
+ * <br>When this event is fired JDA will try to reconnect if possible
+ * unless {@link net.dv8tion.jda.core.JDABuilder#setAutoReconnect(boolean) JDABuilder.setAutoReconnect(Boolean)}
+ * has been provided {@code false}!
+ *
+ * <p>When reconnecting was successful a {@link net.dv8tion.jda.core.events.ReconnectedEvent RecconectEvent} is fired
```

Comment:
ReconnectedEvent, not RecconectEvent.
Also, either ReconnectedEvent or ResumedEvent could be fired after a connection is reestablished.